test/browser/element.vdiff.js

```diff
@@ -39,6 +40,40 @@ const elementTag = defineCE(
 	}
 );
 
+const fixedElementTag = defineCE(
+	class extends LitElement {
+		static get styles() {
+			return css`
+				div {
+					background-color: white;
+					border: 1px solid purple;
+					border-radius: 8px;
+					box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+					box-sizing: border-box;
+					position: fixed;
+				}
+			`;
+		}
+		render() {
+			return html`
+				<div class="vdiff-size">
+					<slot></slot>
+				</div>
+			`;
+		}
+	}
+);
+
+const nestedElementTag = defineCE(
+	class extends LitElement {
+		render() {
+			return html`${unsafeHTML(`
+				<${fixedElementTag} class="vdiff-size"><${elementTag} text="Visual Difference"></${elementTag}></${fixedElementTag}>
```

Comment:
This represents what is likely to happen for consumers of `core`.  If they have a component that wraps `d2l-dialog` for example, they'll add the class to `d2l-dialog`.  vdiff will look for this class, then follow it into `d2l-dialog` to use the element we've added the class to.